### DISCUSSION TITLE
can auth users have both (email and phone) provider?

### DISCUSSION THREAD

**Discussion by JUDE-EQ** [supabase/supabase#28891] [Questions]
is it possible for an auth user to have both (email & phone) provider? meaning email and phone will be verified with OTP so that they can either use `phone` or `email` to login.

**Comment by silentworks** [comment]
Yes this is possible. I've attached a video showing this below:

https://github.com/user-attachments/assets/7f266aba-abd2-448a-8ef0-f9552118a677
